src/clients/noop/index.ts

```diff
@@ -0,0 +1,3 @@
+export function noopDecorator() {
+  return (story: Function) => story;
```

Comment:
The output of Storyshots is `null`. ([this line](https://github.com/tsuyoshiwada/storybook-chrome-screenshot/pull/54/files#diff-2c08c45e6ab1c4cc23b1843350885410R3))
This is due to the implementation of `noopDecorator`.

---

I think that the implementation of `noopDecorator` should be as follows, but what do you think?

```typescript
import { Story } from '../../models/story';

export function noopDecorator() {
  return (storyFn: Function, ctx: Story | undefined) => (
    ctx ? storyFn(ctx) : (context: Story) => storyFn(context)
  );
}
```

Reply:
oops, I missed 2nd arg `ctx: Story | undefined` 😨  You're absolutely right 😂